shepherd_bms/include/compute.h

```diff
@@ -1,6 +1,23 @@
 #ifndef COMPUTE_H
 #define COMPUTE_H
 
-#include "datastructs.h"
+using namespace std;
 
+class ComputeInterface
+{
+    private:
+        int fanSpeed;
+
+    public: 
+        ComputeInterface();
+
+        ~ComputeInterface();
+
+        void enableCharging();
+
+        void setFanSpeed(int newFanSpeed);
+
+        int getPackCurrent();
```

Comment:
return type of int is okay for now, but we might want to change it to a uint8_t later on to save space